src/Glpi/Error/ErrorUtils.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Glpi\Error;
+
+class ErrorUtils
+{
+    /**
+     * @param string $message
+     *
+     * Replace GLPI_ROOT path by '.' in $message
+     * @return string
```

Comment:
We may want to clean more paths in the future. No need to be too precise in the method description.

```suggestion
     * Rewrite file paths to not expose their full filesystem path, as it can be considered as a sensitive information.
     *
     * @param string $message
     * @return string
```